src/transformer/utils/get_verb.php

```diff
@@ -49,6 +51,20 @@ function get_verb($verb, array $config, $lang) {
             ];
             break;
 
+        case 'loggedin':
+            $output = [
+                'id' => 'https://brindlewaye.com/xAPITerms/verbs/loggedin/',
+                'display' => [
+                    $lang = 'logged into'
+                ]
+            ];
+
+            // JISC specific verb id
+            if (utils\is_enabled_config($config, 'send_jisc_data')) {
```

Comment:
we can use a simple function call here only because that one is in the same namespace.

is_enabled_config($config, 'send_jisc_data')